debian/changelog

```diff
@@ -1,3 +1,9 @@
+pg-auto-failover (1.4.2-1) stable; urgency=low
```

Comment:
```suggestion
pg-auto-failover (1.4.2-2) stable; urgency=low
```

Let's bump the fancy version. Maybe we can add a changelog item as well

Reply:
I removed the unnecessary entry because there is no need to release for debian